L/libwebsockets/build_tarballs.jl

```diff
@@ -39,8 +48,10 @@ products = [
 # Dependencies that must be installed before this package can be built
 dependencies = [
     Dependency("OpenSSL_jll"; compat="3.0.16"),
-    Dependency("Zlib_jll")
+    Dependency("Zlib_jll"),
+    Dependency("LibUV_jll")
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6", preferred_gcc_version = v"5.2.0")
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6", preferred_gcc_version = v"15.2.0")
+
```

Comment:
There's quite a bit of difference between gcc 5.2 and 15.2:
* in general we want to use the ***oldest*** gcc version possible to maximise compatibility,
* we don't even have gcc 15 in our toolchains

Also, now there's an extra newline at the end of the file.  I only asked not to remove the newline at the end of the last line, not to add an extra one.
```suggestion
build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6", preferred_gcc_version = v"5.2.0")
```

Reply:
ok, got it. done